@here/harp-lrucache/lib/LRUCache.ts

```diff
@@ -80,7 +85,8 @@ export class LRUCache<Key, Value> {
     /**
      * Iterates over all items from the most recently used item to the least recently used one.
      *
-     * **Note**: Results are undefined if the cache is modified during iteration.
+     * **Note**: Results are undefined if the entire cache is modified during iteration. You may
+     * althought modify the current element in [[callbackfn]] function.
```

Comment:
```suggestion
     * although modify the current element in [[callbackfn]] function.
```

Reply:
Fixed :), finally I have installed Code Spell Checker, so should limit the number of such "flowers"....